### PR TITLE
glide.yaml: Update ct and Ignition

### LINKED ISSUE #606: Update Ignition and CT version
There have been several releases we might pull in.

### PR DESCRIPTION
**Pull request by dghubble**
* Fix calls to container-linux-config-transpiler methods which changed
* Update container-linux-config-transpiler to 2458180e
* Update ignition to v0.17.2

Closes #606

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -19,13 +19,13 @@ import:
   - transport
 # Container Linux Config Transpiler and Ignition
 - package: github.com/coreos/container-linux-config-transpiler
-  version: v0.2.2
+  version: 2458180e0884a43c485d0980f8db62d171a40dd4
```
Comment (dgonyeo) [inline glide.yaml]: Maybe we should make a release if there are unreleased changes you want in here

**Comment by ajeddeloh** [comment]
I'm not up to date with all the changes in clct or ignition, but the changes to handle passing the ast LGTM.

File: glide.yaml (hunk 1 above)
Comment (dghubble) [inline glide.yaml]: I'd be all for that

Comment (dghubble) [inline glide.yaml]: Updated to v0.4.2 tag

**Pull request merged**